user/gamemode/files/gamemoded.user

```diff
@@ -0,0 +1,4 @@
+type = process
```

Comment:
What's the point of a `_gamemode` user if `gamemoded` will run as a user service?

Reply:
```
ERROR: External process failed with exit code 127
ERROR: Output was: 
ERROR: Failed to update cpu governor policy
Requesting update of split_lock_mitigate to 0
Error executing command as another user: Not authorized
``

Reply:
> What's the point of a _gamemode user if gamemoded will run as a user service? 

Again, arguably more important values such as the CPU governor and split lock mitigation are still not used.